zebra-consensus/src/verify/block.rs

```diff
@@ -471,4 +492,71 @@ mod tests {
 
         Ok(())
     }
+
+    #[tokio::test]
+    #[spandoc::spandoc]
+    async fn coinbase() -> Result<(), Report> {
+        install_tracing();
+
+        // Service variables
+        let state_service = Box::new(zebra_state::in_memory::init());
+        let mut block_verifier = super::init(state_service);
+
+        // Get a header of a block
+        let header =
+            BlockHeader::zcash_deserialize(&zebra_test::vectors::DUMMY_HEADER[..]).unwrap();
+
+        let ready_verifier_service = block_verifier.ready_and().await.map_err(|e| eyre!(e))?;
+
+        /// Test 1: Empty transaction
+        let block = Block {
+            header,
+            transactions: Vec::new(),
+        };
+
+        /// Error: no coinbase transaction in block
```

Comment:
this should be an `expect_err` instead of an `unwrap_err` and a spandoc span

Reply:
`expect_err` added, spandocs related moved to https://github.com/ZcashFoundation/zebra/issues/609